src/services/mail/MockMailService.ts

```diff
@@ -0,0 +1,16 @@
+import {RootLog as logger} from '../../utils/RootLogger';
+
+import {MailService} from './MailService';
+
+export class MockMailService implements MailService {
+  constructor() {
+    logger.info('MockMailService: initializing');
+  }
+
+
+  sendMail(senderEmail: string, recipientEmail: string, messageType: string): Promise<any> {
```

Comment:
@defpearlpilot Made those changes. Except I left the 'testSendMail' route in place. It's kinda dumb, but it's the only way to hand test the functionality at the moment, I think. We definitely want to remove it soon, though.